spec/javascript/ParkShowTest.js

```diff
@@ -70,7 +71,8 @@ describe('Amusement Parks Show', () => {
           phone_number: '(413) 786-9300',
```

Comment:
Just like this test, the `ReviewFormContainer` and `ReviewsContainer` need to be tested to ensure that they not only exist, but are passed the right props. Shouldnt take too too long.